builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -315,7 +316,6 @@ node() {
               createSubscriptionFilters(stackName, configLoader.AWS.REGION, roleId);
             }
 
-            updateApiDocUrl(current_environment)
```

Comment:
Are you sure we dont need this method ??

Reply:
we need it. Included in switch case.